[CI] StatelessCoordinationTests testAckListenerReceivesNacksIfPublicationTimesOut failing

**Build Scans:**
- [elasticsearch-periodic #12460 / encryption-at-rest](https://gradle-enterprise.elastic.co/s/7t5tearhylcrm)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:stateless:test" --tests "org.elasticsearch.xpack.stateless.cluster.coordination.StatelessCoordinationTests.testAckListenerReceivesNacksIfPublicationTimesOut" -Dtests.seed=46F7410F1517E405 -Dtests.locale=se-NO -Dtests.timezone=Asia/Phnom_Penh -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.stateless.cluster.coordination.StatelessCoordinationTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testAckListenerReceivesNacksIfPublicationTimesOut'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: expected eventual nack from {node1}{iLIrCAAAQACxpQsIAAAAAA}{0.0.0.0}{0.0.0.0:10}{IScdfhilmrstw}{9.5.0}{8000099-9104000}
```

**Issue Reasons:**
- [main] 1 consecutive failures in test testAckListenerReceivesNacksIfPublicationTimesOut
- [main] 1 consecutive failures in step encryption-at-rest
- [main] 1 consecutive failures in pipeline elasticsearch-periodic

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -692,9 +692,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.CsvIT
   method: test {csv-spec:inlinestats.afterWhere}
   issue: https://github.com/elastic/elasticsearch/issues/152591
-- class: org.elasticsearch.xpack.stateless.cluster.coordination.StatelessCoordinationTests
-  method: testAckListenerReceivesNacksIfPublicationTimesOut
-  issue: https://github.com/elastic/elasticsearch/issues/152605
 - class: org.elasticsearch.xpack.logsdb.RandomizedRollingUpgradeIT
   method: testIndexingStandardSource
   issue: https://github.com/elastic/elasticsearch/issues/152643

--- a/server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java
+++ b/server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java
@@ -680,7 +680,7 @@ public class CoordinatorTests extends AbstractCoordinatorTestCase {
 
                 while (leader.deliverBlackholedRequests()) {
                     // two tasks: (i) deliver the error response, and then (ii) handle it
-                    cluster.runFor(DEFAULT_DELAY_VARIABILITY * 2, "processing blackholed cluster state update requests");
+                    cluster.runFor(DEFAULT_DELAY_VARIABILITY * 2 + 1, "processing blackholed cluster state update requests");
                 }
 
                 assertTrue("expected eventual nack from " + follower0, ackCollector.hasAckedUnsuccessfully(follower0));

